packages/react/src/ChoiceGroupOption.ts

```diff
@@ -0,0 +1 @@
+export * from 'office-ui-fabric-react/lib/ChoiceGroupOption';
```

Comment:
Let's get rid of this file in the new version. ChoiceGroupOption is also exported from the ChoiceGroup file, and having it in two places causes issues like #11534.

Reply:
That would be a breaking change.